src/app/features/addresses/components/molecules/AddressMenu/AddressMenu.tsx

```diff
@@ -43,22 +44,28 @@ const routes = [
   "/adres/:bagId/zaken/"
 ]
 
-const AddressMenu: React.FC<Props> = ({ bagId }) =>
-  // TODO: Read page title, routing, icons from global config JSON
-  <Menu>
-    <Ul>
-      { routes.map(route => {
-          const page = routesObject[route]
-          if (page?.icon === undefined || page?.title === undefined) return null
-          return (
-            <Li key={ route }>
-              <Div>
-                <NavBlock to={ to(route, { bagId }) } icon={ page.icon } header={ page.title } />
-              </Div>
-            </Li>
-          )
-        })
-      }
-    </Ul>
-  </Menu>
+const AddressMenu: React.FC<Props> = ({ bagId }) => {
+  const { data: permitDetails } = usePermitDetails(bagId)
+  const permitDetailsCount = permitDetails?.length
+  const counts = [undefined, undefined, permitDetailsCount]
```

Comment:
Does this array have to match the order of the Navblocks? Isn't that a bit tricky?

Reply:
Yes, but it's the most "dynamic" solution I could come up with, for now. Because hooks can't be called in a loop. Let's see if we can improve on this, when the other counter endpoints are also available.